Sources/Action/Action+Extensions.swift

```diff
@@ -21,3 +21,14 @@ public extension Action where Input == Void {
         return execute(())
     }
 }
+
+public extension CompletableAction {
+    /// Emits everytime work factory completes.
+    var completions: Observable<Void> {
```

Comment:
Just wondering in general - what's the use case to know when _any_ work completed? Given you don't know exactly which did? I'm wondering what this PR tries to solve

Reply:
@freak4pc This is actually very useful. It's not just a syntactic sugar. Sometimes you want to know exactly when an `Action` ~> `isExecuting` ~> `didFinishExecuting`  So this emits a sequence to let know hey I am done with what that long operation now you can do another thing in the next step. You can even use this to chain operations. etc.